generator/lookml.py

```diff
@@ -126,29 +130,51 @@ def _lookml(namespaces, glean_apps, target_dir, namespace_filter=[]):
 
             logging.info("  Generating views")
             v1_name: Optional[str] = v1_mapping.get(namespace)
-            for view_path in _generate_views(client, view_dir, views, v1_name):
-                logging.info(f"    ...Generating {view_path}")
+            for view in views:
+                views_with_v1_name.append(
+                    partial(_generate_view, client, view_dir, view, v1_name)
+                )
+
+    with ThreadPool(parallelism) as pool:
+        pool.map(lambda x: x(), views_with_v1_name, chunksize=1)
```

Comment:
It'd be nice to have a comment here explaining why `pool.map()` with `chunksize=1` was called for, compared to the other cases which just use `pool.starmap()`.

Reply:
In this case `starmap` would actually be slightly more readable